File: angular/src/app/main/finance/shared/dto/arinvd-dto.ts
export class ARINVDDto {
    id: number;
    detID: number;
    accountID: string;
    subAccID: number | null;
    docNo: number | null;
    invNumber: string;
    invAmount: number | null;
    taxAmount: string;
    recpAmount: number | null;
    chequeNo: string;
    adjust: boolean;
    narration: string;
}

File: aspnet-core/src/ERP.Application.Shared/AccountReceivables/RouteInvoices/Dtos/ARINVDDto.cs

using System;
using Abp.Application.Services.Dto;

namespace ERP.AccountReceivables.RouteInvoices.Dtos
{
    public class ARINVDDto : EntityDto
    {
		public int DetID { get; set; }

		public string AccountID { get; set; }

		public int? SubAccID { get; set; }

		public int? DocNo { get; set; }

		public string InvNumber { get; set; }

		public double? InvAmount { get; set; }

		public double? TaxAmount { get; set; }

		public double? RecpAmount { get; set; }

		public string ChequeNo { get; set; }

		public bool Adjust { get; set; }

		public string Narration { get; set; }

        public string AccountName { get; set; }

        public string SubAccName { get; set; }



    }
}